x-pack/qa/security-tools-tests/src/test/java/org/elasticsearch/xpack/security/enrollment/tool/CreateEnrollmentTokenToolTests.java

```diff
@@ -151,14 +150,15 @@ public static void closeJimfs() throws IOException {
         }
     }
 
-    public void testCreateToken() throws Exception {
-        String scope = randomBoolean() ? "node" : "kibana";
-        String output = execute("--scope", scope);
-        if (scope.equals("kibana")) {
-            assertThat(output, containsString("1WXzQ4eVlYOngzWXFVX3JxUXdtLUVTcmtFeGNuT2cifQ=="));
```

Comment:
This was checking the end of the generated base64 to see if it contains the appropriate API key. These broke because they're at the end, and the length of the base64'd data changed. I changed these to explicitly base64 decode and check, which should be much more robust.